lib/ucfg.rb

```diff
@@ -43,20 +43,27 @@ def self.validate_recursively(config, schema, config_path)
     # fails if property is provided as other type
     config.each do |key, value|
       if schema["properties"].key?(key) && schema["properties"][key].key?("type")
-        if schema["properties"][key]["type"] != value_type(value)
+        if schema["properties"][key]["type"].include?(value_type(value))
```

Comment:
Initially I was a bit surprised this worked but I can see why now: both strings and arrays have an `include?` method. I have modified one of the specs so this branch is currently failing - let me know if it is not obvious what you need to change.

Reply:
Done